components/Collection.vue

```diff
@@ -16,10 +16,9 @@
 
 		<slot name="before-description" v-bind="$props"></slot>
 
-		<section class="description" v-if="stac.description">
-			<h3>Description</h3>
-
-			<Description :description="stac.description"></Description>
+		<section class="description" v-if="stac.description || stac.deprecated || supportedBy || affectedByMissing">
+			<h3 v-if="stac.description">Description</h3>
+			<Description v-if="stac.description" :description="stac.description"></Description>
```

Comment:
```suggestion
			<template v-if="stac.description">
				<h3>Description</h3>
				<Description :description="stac.description"></Description>
			</template>
```

Very small thing, but I think this makes it a little easier to understand the scope of the blocks/conditions.

Reply:
Done! https://github.com/Open-EO/openeo-vue-components/pull/107/commits/56f0f24d138cc408b00bbf2c7edc6cee0d7dfc33